test/test_3d.jl

```diff
@@ -102,6 +103,32 @@ end
     end
   end
 
+  @testset "Parallel Trixi.jl" begin
+    # Clean output directory
+    isdir(outdir) && rm(outdir, recursive=true)
+
+    # Run trixi in parallel with 2 ranks
+    mpiexec() do cmd
+      run(`$cmd -n 2 $(Base.julia_cmd()) --threads=1 --project=@. -e 'using Trixi; trixi_include(@__MODULE__, joinpath(examples_dir(), "p4est_3d_dgsem", "elixir_mhd_alfven_wave_nonconforming.jl"), trees_per_dimension=(1, 1, 1),  maxiters=4)'`)
+    end
```

Comment:
What would be the right way to run Trixi in parallel from a Trixi2Vtk test, @sloede?
I'm getting the error:
```
ERROR: ERROR: ArgumentError: Package Trixi [a7f1ee26-1774-49b1-8366-f1abc58fbfcb] is required but does not seem to be installed:
``` 
... And I don't know how to fix it.

Reply:
I haven't checked, but is this the same way we run the MPI tests in Trixi.jl?